svelte/src/libs/stores/pkgs.ts

```diff
@@ -166,6 +166,10 @@ const init = async function () {
     await refreshPackages();
     await monitorTeaDir();
     initialized = true;
+
+    setInterval(async () => {
+      await refreshPackages();
+    }, 1000 * 60 * 30);
```

Comment:
in line 247 there is a refresh, did you figure out why it was not working?

Reply:
it looks like it is only a timeout, so maybe that's why?

Reply:
I didn't notice that, I'm not really sure.  Maybe it's best if someone else can verify they have the same issue first.  For me, I just notice that if I don't restart my tea gui then I don't see new updates

Reply:
I'm not really familiar with svelete.  Is it possible ondestroy is called which would prevent the fresh from actually refreshing packages?

Reply:
ill dig deeper into this today but @ABevier  implemeted the refresh strategy here.